.github/workflows/build-release.yml

```diff
@@ -0,0 +1,36 @@
+name: build-release
+run-name: ${{ github.workflow }} is to create release wheel file for pypi
+on:
+  push:
+    branches:
+      - 'release/[0-9]+.[0-9]+'
+    workflow_dispatch:
```

Comment:
Do we want to build a release wheel on each push to release branch or only when tags are created (i.e when a release is published on github)?